plugins/signals-backend/src/service/router.ts

```diff
@@ -42,6 +44,7 @@ export interface RouterOptions {
   discovery: PluginEndpointDiscovery;
   auth?: AuthService;
   userInfo?: UserInfoService;
+  permissions?: PermissionEvaluator;
```

Comment:
Yeah you can search for PermissionEvaluator throughout and make it PermissionsService instead; it occurs in a couple of places.

Note that a couple lines up, we could have `IdentityService` and `DiscoveryService` too